esmvaltool/utils/cmorizers/obs/cmorize_obs_Landschuetzer2016.py

```diff
@@ -80,10 +84,14 @@ def extract_variable(var_info, raw_info, out_dir, attrs):
             _fix_coords(cube)
             _fix_data(cube, var)
             _add_metadata(cube, attrs)
-            _save_variable(cube, var, out_dir, attrs,
-                           fill_value=cube.data.fill_value,
```

Comment:
@bouweandela  Without passing `fill_value` to iris, the variable in the netcdf file is saved without the `_FillValue` attribute, as there is some glitch in iris. 

This is a workaround suggested here https://github.com/ESMValGroup/ESMValTool/issues/302#issuecomment-381099386 by @valeriupredoi 

I guess that the operation `cube.data.fill_value` realizes the cube, that's why you removed it?

Reply:
indeed, this is not a glitch but a feature in iris 2+ https://scitools.org.uk/iris/docs/latest/whitepapers/missing_data_handling.html

Reply:
Indeed, that realizes the data. Thank you for the link to the discussion, apparently 1e20 as a fill value comes from the cmor tables. I have hardcoded it for now, as I think that is the simplest solution. If you do not like the hardcoded version, feel free to read to add code to read it from the cmor table.

Reply:
I got to assume that there is no way to retrieve the missing value from the cube without realizing it ... so the only way is to hardcode it in the cmorizer ..

btw, is it a big issue to realize the cube within the call of iris.save? Something like 
```python
iris.save(cube, file_path, fill_value=cube.data.fill_value, **kwargs)
```

In this way the missing value is set coherently to the cube's data and it will be used by all python cmorizers, by also avoiding to specifically hardcode it.

Reply:
Note that using the default fill value works absolutely fine, but I guess it is not CMOR compliant.

Reply:
@bouweandela  @valeriupredoi I probably miss this, but setting the fill_value within `iris.save`
```python
    iris.save(cube, file_path, fill_value=1e20, **kwargs)
 ```
will also modify the fill_value data written to the netcdf file?

Reply:
yes, and 1e20f is the CMOR correct one

Reply:
> is it a big issue to realize the cube

Because in this case the data is not that much anyway, it's not a big issue to realize the data, it will just result in a slighty slower run with slightly higher memory use. This may be different for other datasets, so I prefer to set a good example.

> In this way the missing value is set coherently to the cube's data and it will be used by all python cmorizers, by also avoiding to specifically hardcode it.

The fill value should be 1e20 according to the CMOR table, so setting it to something else is wrong. If you want to avoid hardcoding, read it from the CMOR table.

> setting the fill_value within iris.save will also modify the fill_value data written to the netcdf file?

Setting the fill value in iris.save works as you would expect, just try it or read the iris documentation.

Reply:
That's fine. I don't see any other issue.